hedera-mirror-web3/src/main/java/com/hedera/services/store/models/Token.java

```diff
@@ -1840,7 +1842,8 @@ public Token setLastUsedSerialNumber(final long lastUsedSerialNumber) {
     }
 
     public List<CustomFee> getCustomFees() {
-        return customFees;
+        return Collections.unmodifiableList(
+                customFees != null && customFees.get() != null ? customFees.get() : new ArrayList<>());
```

Comment:
`customFees.get()` should never be null per `TokenDatabaseAccessor.getCustomFees()`.

```suggestion
        return customFees != null ? Collections.unmodifiableList(customFees.get()) : Collections.emptyList();
```